tests/ratingDriver.py

```diff
@@ -0,0 +1,162 @@
+
+from selenium.webdriver import Chrome
+from loginDriver import logIn
+
+import time
+
+
+
+class rating:
+    def __init__(self):
+        self.driver = Chrome()
+    
+    def Close(self):
+        self.driver.quit()
+    
+    def getTimeCreationOfEvaluation(self):
+        html = self.driver.page_source        
+        str1 = "grade by"
+        a = html.find(str1)
+        timeCreation = html[(a-21):(a-2)]
+        return timeCreation
+    
+    def usernameToCssUsername(username):
```

Comment:
figure out which methods are **_private_** (only called by other code in the same class) and which methods are **_public_** (called by code in other classes). The private methods should start with one leading underscore (e.g., _usernameToCssUsername, although note that when you fix all methods naming to lowercase/underscore, it will end up as _username_to_css_username).

Reply:
Use the refactoring tools in your IDE to do this automatically.

Reply:
This is no longer an issue in the test suite.